docker/install/ubuntu_install_androidsdk.sh

```diff
@@ -68,20 +67,20 @@ extras;google;simulators
 extras;google;webdriver
 extras;m2repository;com;android;support;constraint;constraint-layout;1.0.2
 extras;m2repository;com;android;support;constraint;constraint-layout-solver;1.0.2
-lldb;2.3
 platforms;android-26
 platforms;android-27
 tools
-ndk-bundle
+ndk;21.3.6528147
```

Comment:
`ndk-bundle` install the latest NDK version.
I specified the currently latest version.